packages/server/postgres/init.sql

```diff
@@ -0,0 +1,2 @@
+DROP DATABASE IF EXISTS web;
```

Comment:
will this drop the DB if the image gets rebuilt? i think if we specify a POSTGRES_DB env var then we can get rid of this file & have it make the DB for us

Reply:
yeah good idea